openapi.yaml

```diff
@@ -262,12 +266,16 @@ paths:
                         text:
                           description: The translated text.
                           type: string
+                        billed_characters: 
+                          description: Optional. Number of characters counted by DeepL for billing purposes.
```

Comment:
I think "Optional" here is misleading, because it is a response field. Better would be "Only present if the show_billed_characters parameter is true".